src/olympia/addons/migrations/0019_auto_20200805_1350.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.15 on 2020-08-05 13:50
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("addons", "0018_auto_20200803_1311"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="addonguid",
+            name="hashed_guid",
+            field=models.CharField(max_length=100, null=True),
```

Comment:
I thought the problem occurred if the guid length was above 80?

Reply:
actually, isn't the hash always 64 chars long?

Reply:
correct, fixed.